b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/role/decodePatchRole.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import callPatchRole from
+    "../../../apiCall/settings/role/callPatchRole";
+import { commonDecode } from "../../../util/apiUtil/commonDecode";
+import { PatchMethod } from "../../../util/common/common";
+
+function getAddReplaceBody(patchMethod, path, value) {
+    return {
+        "operations": [
+            {
+                "op": patchMethod,
+                "path": path,
+                "value": value
+            }
+        ]
+    }
+}
+
+function getRemoveBody(patchMethod, path, value) {
+    return {
+        "operations": [
+            {
+                "op": patchMethod,
+                "path": `${path}[value eq ${value}]`
+            }
+        ]
+    }
+}
+
+function getPatchBody(patchMethod, path, value) {
+    switch (patchMethod) {
+        case PatchMethod.ADD:
+
+            return getAddReplaceBody(patchMethod, path, value);
+
+        case PatchMethod.REPLACE:
+
+            return getAddReplaceBody(patchMethod, path, value);
+        case PatchMethod.REMOVE:
+
+            return getRemoveBody(patchMethod, path, value);
+    }
+}
+
+/**
+ * 
+ * @param {*} session - session
+ * @param {*} roleUri - uri of the role
+ * @param {*} patchMethod - `PatchMethod` value
+ * @param {*} path - path variable (`users` | `permission` | `displayName`)
+ * @param {*} value - value that need to be updated. If `patchMethod` is `PatchMethod.ADD` then `value` 
+ * should be an `array`
```

Comment:
Shal we remove these asterisks?

Reply:
Fixed when the functionality moved to a different location 
https://github.com/Achintha444/samples-is/commit/b8a143e14872c6b7a91d0f029bfffffa5fea66ef#diff-e70752c7d2a32606406568ac366fe30cf2f7c3ce42ec873152e4f583c38e394f